### ISSUE TITLE
How to publish aab?

### ISSUE THREAD

**Issue by JustickDM** [DamianAntonowicz/CakeAndXamarinPlayground#4]

**Comment by DamianAntonowicz** [comment]
You need to select AAB from Android Package Format dropdown in your Android project settings (more info here: https://devblogs.microsoft.com/xamarin/android-app-bundle/). Then Cake task "PublishAPK" will publish AAB instead of APK. However my script is not adjusted to publishing AAB and it will fail at some point.

**Comment by JustickDM** [comment]
Yep, this script doest work for publishing AAB. Can you modify the script?

**Comment by DamianAntonowicz** [comment]
Unfortunately adjusting this Cake script to publish AAB is not on my roadmap for near future. However I will add this to my to do list.

**Comment by JustickDM** [comment]
APK no metter already for publishing (for new apps) on google play since from 08.2021
